pluginconfig/commit_test.go

```diff
@@ -211,10 +162,20 @@ func TestCommitOffchainConfig_Validate(t *testing.T) {
 			},
 			true,
 		},
+		{
+			"invalid, new msg scan batch size not provided",
+			fields{
+				RemoteGasPriceBatchWriteFrequency: *commonconfig.MustNewDuration(1),
+				TokenPriceBatchWriteFrequency:     *commonconfig.MustNewDuration(1),
+				PriceSources:                      map[types.Account]ArbitrumPriceSource{},
+			},
+			true,
+		},
 	}
```

Comment:
Add tests for sync durations and timeouts?

Reply:
Sync duration and timeout have defaults that are applied in the plugin constructors, so I think its OK if they're zero.

Although there could be an argument for not having these defaults since its less visible...

Reply:
Then will rip out the defaults and enforce that they're specified in the offchain config